uaaextras/webapp.py

```diff
@@ -448,16 +453,13 @@ def signup():
             return render_template('signup.html')
         # Check for feds here
         valid_gov_email = False
-        for row in FED_DOTGOV_LIST:
-            if re.search(row[0].replace('.', '\.') + '$', email, flags=re.I):
-                valid_gov_email = True
-        if re.search('\.mil$', email, flags=re.I):
-            valid_gov_email = True
-        if re.search('\.fed\.us$', email, flags=re.I):
+        if app.config['VALID_FED_DOMAINS_REGEX'].search(email):
```

Comment:
This actually makes more sense. I totally forgot about `endswith` taking a tuple. Was originally using regex over string.find for speed.